contactform.php

```diff
@@ -46,8 +46,8 @@ public function __construct()
         parent::__construct();
 
         $this->displayName = $this->trans('Contact form', array(), 'Modules.Contactform.Admin');
-        $this->description = $this->trans('Description for contact form module', array(), 'Modules.Contactform.Admin');
-        $this->ps_versions_compliancy = array('min' => '1.7.1.0', 'max' => _PS_VERSION_);
+        $this->description = $this->trans('Adds a contact form to the "Contact us" page.', array(), 'Modules.Contactform.Admin');
+        $this->ps_versions_compliancy = array('min' => '1.7.2.0', 'max' => _PS_VERSION_);
```

Comment:
Why do you want to change the min version for the module ? Just because the description is changing ?